### PR TITLE
docs: apply Menu Engineering to "How-to" pages

### PR DESCRIPTION
**Pull request by linostar**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

Apply Menu Engineering to "How-to" pages, and use links in landing pages where possible.
Refer to [this example](https://docs.google.com/document/d/1T2aiuVCsF5krBQdqt8Rfaq3gXzhxaypoyetkCSN8RL4/edit#heading=h.5p2vvzpgp6ck) for guidance.

### CODE REVIEW

**Comment by linostar** [comment]
@cjdcordeiro @tigarmo Ready for review.

File: docs/how-to/index.rst [hunk 1]
```diff
@@ -14,13 +14,39 @@ adapt the steps to fit your specific requirements.
...
-   Convert an entrypoint to a Pebble layer <convert-to-pebble-layer>
-   Publish a rock to a registry <publish-a-rock.rst>
-   Migrate a Docker image to a chiselled rock <migrate-to-chiselled-rock.rst>
+
+Chisel
+------
+
+Chisel is a tool that lets you keep only the required files from one or more apt
+packages and discard the rest. Upgrade your Chisel slicing skills by learning
+how to cut, install and publish slices.
```
Comment (cjdcordeiro) [inline docs/how-to/index.rst:23]:
```suggestion
Upgrade your Chisel slicing skills by learning
how to cut, install and publish slices.
```

Let's leave the explanations for the explanations page...it's less error-prone this way

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
The content lgtm. I like the end result.

I wonder if the folder re-organization is needed though...

**Review by tigarmo — APPROVED** [review]
Looks pretty good to me! I also like the folder-reorg, it was starting to get very crowded.

**Comment by tigarmo** [comment]
@linostar can you fix the conflict?

**Comment by linostar** [comment]
> @linostar can you fix the conflict?

in UK English: done, mate.
in US English: conflict fixer is my middle name.

**Comment by linostar** [comment]
What's this error in the snap tests? new repo policy?

```
<11>Mar 19 13:05:16 ubuntu: This job has failed to pass a repository policy compliance check as defined in the https://github.com/canonical/repo-policy-compliance repository. The specific failure is listed below. Please update the settings on this project to fix the relevant policy.
<11>Mar 19 13:05:16 ubuntu: authorization comment not found on PR, expected: /canonical/self-hosted-runners/run-workflows 9aea50e9078fb72944c00119113dc052f91dd4a2 - execution not authorized, a comment from a user with write permission or above on the repository approving the workflow was not found on a PR from a fork, the comment should include the string '/canonical/self-hosted-runners/run-workflows <commit SHA>' where the commit SHA is the SHA of the latest commit on the branch, branch_name='ROCKS-1023/docs/apply-menu-engineering-to-howto-pages', commit_sha='9aea50e9078fb72944c00119113dc052f91dd4a2', pull_for_branch.number=507
<11>Mar 19 13:05:16 ubuntu: curl: (22) The requested URL returned error: 403
<11>Mar 19 13:05:16 ubuntu: Stopping execution of jobs due to repository setup is not compliant with policies.
<11>Mar 19 13:05:16 ubuntu: This runner will be stopped or lost, please fix the setup of the repository, then rerun this job.
```

**Review by lengau — APPROVED** [review]
Looks good, thanks!

**Pull request merged**